### ISSUE TITLE
bin/pyopenvdb start error: "No rule to make target 'Grid.cc', needed by 'Grid.o'"

### ISSUE THREAD

**Issue by deehrlic** [theNewFlesh/docker_pyopenvdb#3]
Hi, trying to set up this lib on my Windows 10 WSL2 setup and got Docker working and all the steps in the bin/pyopenvdb start process to go save for the last one, which appears to be complaining about the Makefile. Any advice on what to do? Attaching terminal output below, let me know if you need any other info. Thanks!

```Building pyopenvdb
[+] Building 16.3s (21/21) FINISHED
 => [internal] load build definition from pyopenvdb.dockerfile                                                     0.0s
 => => transferring dockerfile: 48B                                                                                0.0s
 => [internal] load .dockerignore                                                                                  0.0s
 => => transferring context: 2B                                                                                    0.0s
 => [internal] load metadata for docker.io/library/ubuntu:20.04                                                   10.6s
 => [ 1/16] FROM docker.io/library/ubuntu:20.04@sha256:3c9c713e0979e9bd6061ed52ac1e9e1f246c9495aa063619d9d695fb80  0.0s
 => [internal] load build context                                                                                  0.0s
 => => transferring context: 33.44kB                                                                               0.0s
 => https://dl.bintray.com/boostorg/release/1.68.0/source/boost_1_68_0.tar.gz                                      5.3s
 => CACHED [ 2/16] WORKDIR /root                                                                                   0.0s
 => CACHED [ 3/16] RUN apt update                                                                                  0.0s
 => CACHED [ 4/16] RUN DEBIAN_FRONTEND="noninteractive" apt-get install -y tzdata                                  0.0s
 => CACHED [ 5/16] RUN echo "\n\033[0;36mINSTALL DEPENDENCIES\033[0m";     apt update &&     apt install -y --fix  0.0s
 => CACHED [ 6/16] COPY dev_requirements.txt /root/dev_requirements.txt                                            0.0s
 => CACHED [ 7/16] RUN echo "\n\033[0;36mINSTALL PYTHON DEPENDENCIES\033[0m";     pip3 install -r /root/dev_requi  0.0s
 => CACHED [ 8/16] ADD https://dl.bintray.com/boostorg/release/1.68.0/source/boost_1_68_0.tar.gz boost.tar.gz      0.0s
 => CACHED [ 9/16] RUN echo "\n\033[0;36mDOWNLOAD BOOST\033[0m";     tar xzf /root/boost.tar.gz &&     echo "usin  0.0s
 => CACHED [10/16] WORKDIR /root/boost_1_68_0                                                                      0.0s
 => CACHED [11/16] RUN echo "\n\033[0;36mBUILD BOOST LIBRARIES\033[0m";      ./bootstrap.sh         --with-python  0.0s
 => CACHED [12/16] WORKDIR /root                                                                                   0.0s
 => CACHED [13/16] RUN echo "\n\033[0;36mCLONE OPENVDB\033[0m";     git clone https://github.com/AcademySoftwareF  0.0s
 => CACHED [14/16] WORKDIR /root/openvdb/openvdb                                                                   0.0s
 => [15/16] COPY Makefile /root/openvdb/openvdb/Makefile                                                           0.1s
 => ERROR [16/16] RUN echo "\n\033[0;36mBUILD PYOPENVDB\033[0m";     make python                                   0.3s
------
 > [16/16] RUN echo "\n\033[0;36mBUILD PYOPENVDB\033[0m";     make python:
#21 0.279
#21 0.279 BUILD PYOPENVDB
#21 0.281 grep: version.h: No such file or directory
#21 0.283 grep: version.h: No such file or directory
#21 0.284 grep: version.h: No such file or directory
#21 0.285 grep: version.h: No such file or directory
#21 0.287 grep: version.h: No such file or directory
#21 0.288 grep: version.h: No such file or directory
#21 0.289 grep: version.h: No such file or directory
#21 0.290 grep: version.h: No such file or directory
#21 0.292 grep: version.h: No such file or directory
#21 0.293 grep: version.h: No such file or directory
#21 0.294 grep: version.h: No such file or directory
#21 0.295 grep: version.h: No such file or directory
#21 0.296 make: *** No rule to make target 'Grid.cc', needed by 'Grid.o'.  Stop.
------
executor failed running [/bin/sh -c echo "\n${CYAN}BUILD PYOPENVDB${NO_COLOR}";     make python]: exit code: 2
ERROR: Service 'pyopenvdb' failed to build```

**Comment by theNewFlesh** [comment]
Hmm....I no longer can get even that far in my docker build. Looks like some apt dependencies changed. To be honest, this project was to pip package pyopenvdb for python3.7 and 3.8. Those packages have long been complete and uploaded to pypi. So, I'm not super interested in revisiting the process of getting the container working again, because it was incredibly painful.

You can either pip install pyopenvdb (`pip3.7 install pyopenvdb`) as in the way its done in pyopenvdb_prod.dockerfile, you can pull the docker image from dockerhub (`docker pull thenewflesh/pyopenvdb:latest`), which should be static upon time of publish, or you can get it to work and submit a PR. Your call.

**Comment by deehrlic** [comment]
Thanks for quick response - still struggling to install the actual lib on WSL2 so it's VM/other OS time I suppose. I have a version of the dockerfile that someone helped me get working save the actual compilation part, so if you want that I can send it over. I appreciate the help!

**Comment by theNewFlesh** [comment]
Yeah that be great

**Comment by deehrlic** [comment]
Here it is - big fix was using newer ubuntu version (20.04) and reordering some stuff to get around a noninteractive prompt created when installing tzdata lib. This file should get through everything save the making part which breaks as mentioned above. Uploaded as txt since it won't let me add a dockerfile file

[pyopenvdb.dockerfile.txt](https://github.com/theNewFlesh/docker_pyopenvdb/files/6289519/pyopenvdb.dockerfile.txt)

**Comment by theNewFlesh** [comment]
I've updated my python3.8 branch to include these changes. Thanks.
